grafast/grafserv/src/servers/hono/v4/index.ts

```diff
@@ -0,0 +1,254 @@
+import type { Context as HonoContext, Hono, MiddlewareHandler } from "hono";
+import type { StatusCode } from "hono/utils/http-status";
+
+import {
+  convertHandlerResultToResult,
+  GrafservBase,
+  makeGraphQLWSConfig,
+  normalizeRequest,
+  processHeaders,
+} from "../../../index.js";
+import type {
+  EventStreamHeandlerResult,
+  GrafservBodyJSON,
+  GrafservConfig,
+  RequestDigest,
+  Result,
+} from "../../../interfaces.js";
+import { GRAPHQL_TRANSPORT_WS_PROTOCOL, makeServer } from "graphql-ws";
+import { UpgradeWebSocket } from "hono/ws";
+import { web } from "webpack";
+
+declare global {
+  namespace Grafast {
+    interface RequestContext {
+      Hono: {
```

Comment:
This should include a version number and I think the others are lower case so it should match that too:

```suggestion
      honov4: {
```